backend/app/assets/stylesheets/spree/backend/components/_pills.scss

```diff
@@ -0,0 +1,43 @@
+.pill {
+  display: inline-block;
+  padding: 2px 10px;
+  border-radius: 10px;
+  font-size: 11px;
```

Comment:
I'm more curious about who the perspective is here. This is a very specific component that I have sized specifically. If users viewing the admin bump up their font size this will scale along with it.

If we decide to change the root font size of the admin we'll need to go through components one by one and figure out what needs to change. I'd normally be happy to use `rem` but our admin root font size is `13px` which I can see changing in the future but I wouldn't want this component to change.